build.sh

```diff
@@ -1,9 +1,13 @@
-#!/usr/bin/env bash
-set -e
+#!/bin/bash
 
-docker build -t aochagavia_rust_raffler aochagavia-rust
-docker build -t shawnmccool_scala_raffler shawnmccool-scala
-docker build -t basbl_zsh_raffler basbl-zsh
-docker build -t lucasvanlierop_cobol_raffler lucasvanlierop-cobol
-docker build -t rjkip_elixir_raffler rjkip-elixir
-docker build -t sgoettschkes_haskell_raffler sgoettschkes-haskell
+dockerfiles=$(ls */Dockerfile)
```

Comment:
The current setup was just to get thing rolling, I was planning to improve it but thanks for fixing it!